lnwallet/btcwallet/btcwallet.go

```diff
@@ -324,6 +324,34 @@ func (b *BtcWallet) Start() error {
 		}
 	}
 
+	// Because we might add new "default" key scopes over time, they are
+	// created correctly for new wallets. Existing wallets don't
+	// automatically add them, we need to do that manually now.
+	for _, scope := range waddrmgr.DefaultKeyScopes {
+		_, err := b.wallet.Manager.FetchScopedKeyManager(scope)
+		if err != nil {
```

Comment:
nit: should we match on `ErrScopeNoteFound` ?

Reply:
+1